izpack-installer/src/main/java/com/izforge/izpack/installer/requirement/LockFileChecker.java

```diff
@@ -123,10 +125,29 @@ public boolean check()
      */
     protected boolean handleLockFile(File file)
     {
-        boolean result = true;
-        logger.fine("Lock File Exists, asking user for permission to proceed.");
         StringBuilder msg = new StringBuilder();
         String appName = installData.getInfo().getAppName();
+        if (Installer.getInstallerMode() == Installer.INSTALLER_AUTO) {
+            logger.fine("Lock File Exists.");
+            msg.append("The " + appName + " installer you are attempting to run seems to have a copy already running.");
+            msg.append(" This could be from a previous failed installation attempt or you may have accidentally");
+            msg.append(" launched the installer twice. The recommended action is to wait for the other copy of the");
+            msg.append(" installer to start. If you are sure there is no other copy of the installer running, you can");
+            msg.append(" delete the lock file \"");
```

Comment:
Do we not need any new lines here? Also if we have the same line feeds as within the GUI, extracting the identical parts into a separate method would make sense.